tests/testsupport/cythonarrayutil.pxi

```diff
@@ -1,9 +1,10 @@
+from __future__ import print_function
```

Comment:
This does not belong here, as ",pxi" file need not be included at the first line of the host, causing assertion to trip if not.